docs/source/configuration/in-memory-caching.mdx

```diff
@@ -1,10 +1,10 @@
 ---
-title: In-Memory Caching in the Apollo Router
+title: In-Memory Caching in the GraphOS Router
```

Comment:
In-Memory Caching is available in the Core Router. Do we still use "GraphOS Router" as a blanket term?

Reply:
Will edit the title and clarify in body it applies to both